Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -0,0 +1,117 @@
+# How to Configure Leap Motion Hand Tracking in MRTK 
+
+A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-controller/) is required to use this data provider.
+
+The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
+
+## Using Leap Motion tracking in MRTK
+1. Prepare MRTK project for Leap Motion
+
+    - This step only applies if the source of MRTK is cloned from the git repo and NOT the unity packages.
```

Comment:
Nit: On formatting, it's a bit weird seeing the double indent (i.e. I get that this "this step only applies" is kinda a note... maybe we should use the NOTE format that is used below?